phase1-coordinator/src/coordinator.rs

```diff
@@ -2199,12 +2199,17 @@ impl Coordinator {
 
     #[inline]
     fn load_current_round_height(storage: &Disk) -> Result<u64, CoordinatorError> {
-        // Fetch the current round height from storage.
-        match storage.get(&Locator::RoundHeight)? {
-            // Case 1 - This is a typical round of the ceremony.
-            Object::RoundHeight(current_round_height) => Ok(current_round_height),
-            // Case 2 - Storage failed to fetch the round height.
-            _ => Err(CoordinatorError::StorageFailed),
+        match storage.exists(&Locator::RoundHeight) {
```

Comment:
this can be smaller indentation with an `if` - `else` statement, but I don't mind, both are just as readable :slightly_smiling_face: